src/amb_kf.c

```diff
@@ -21,6 +21,10 @@
 #include <assert.h>
 #include <cblas.h>
 #include <clapack.h>
+#ifdef __WIN32
+  /*Avoid error of this being redefined in math.h*/
+  #undef abs
+#endif
```

Comment:
Could you instead move this guard into the offending `f2c.h` headers that are included by LAPACK/BLAS?

https://github.com/swift-nav/libswiftnav/blob/master/clapack-3.2.1-CMAKE/F2CLIBS/libf2c/f2c.h#L160
https://github.com/swift-nav/libswiftnav/blob/master/clapack-3.2.1-CMAKE/INCLUDE/f2c.h#L160